<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>seleccion_tipo_poliza</class>
 <widget class="QDialog" name="seleccion_tipo_poliza">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1050</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1050</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ingresar Datos:</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="btnVolver">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>620</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 153, 255);
border-radius: 8px;
</string>
   </property>
   <property name="text">
    <string>Volver</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfirmar">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>620</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 153, 255);
border-radius: 8px;
</string>
   </property>
   <property name="text">
    <string>Confirmar</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>951</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(209, 236, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>831</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 232, 163);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nro. de Cliente:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo de Documento</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Datos del Cliente:</string>
     </property>
    </widget>
    <widget class="QLabel" name="txtNombre">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>XXXXXXXXXXXXX</string>
     </property>
    </widget>
    <widget class="QLabel" name="txtNumeroCliente">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>99-999999999999</string>
     </property>
    </widget>
    <widget class="QLabel" name="txtTipoDocumento">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>XXXXXXXXXXXXX</string>
     </property>
    </widget>
    <widget class="QLabel" name="txtApellido">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>XXXXXXXXXXXXX</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>70</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Domicilio:</string>
     </property>
    </widget>
    <widget class="QLabel" name="txtNumeroDocumento">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>100</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>99-999999999999</string>
     </property>
    </widget>
    <widget class="QLabel" name="txtDomicilio">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>70</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>XXXXXXXXXXXXX</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>100</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nro. de Documento</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>40</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Apellido:</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>911</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="txtFechaInicioVigencia">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>70</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0) ;
</string>
     </property>
     <property name="placeholderText">
      <string>dd/mm/aaaa</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 232, 163);
border-radius: 8px;</string>
     </property>
     <property name="text">
      <string>   Responsabilidad Civil</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>35</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fecha de inicio de vigencia de la póliza:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbFormaPago">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>160</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>--- Seleccione una opción</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mensual</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Semestral</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 232, 163);
border-radius: 8px;</string>
     </property>
     <property name="text">
      <string>   Resp. Civil, Robo o incendio total</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 232, 163);
border-radius: 8px;</string>
     </property>
     <property name="text">
      <string>   Todo total</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>210</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 232, 163);
border-radius: 8px;</string>
     </property>
     <property name="text">
      <string>   Terceros Completos</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>270</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 232, 163);
border-radius: 8px;</string>
     </property>
     <property name="text">
      <string>   Todo Riesgo con Franquicia</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Forma de pago:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbRespCivil">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbIncendioTotel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbTodoTotal">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbTercerosCompletos">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>220</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbTodoRiesgo">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>280</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_7">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-40</y>
     <width>1071</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 153, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Alta de Póliza</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>669</y>
     <width>1071</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 153, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semibold</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ingresar Datos:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>rbRespCivil</tabstop>
  <tabstop>rbIncendioTotel</tabstop>
  <tabstop>rbTodoTotal</tabstop>
  <tabstop>rbTercerosCompletos</tabstop>
  <tabstop>rbTodoRiesgo</tabstop>
  <tabstop>txtFechaInicioVigencia</tabstop>
  <tabstop>cbFormaPago</tabstop>
  <tabstop>btnVolver</tabstop>
  <tabstop>btnConfirmar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
